### ISSUE TITLE
[Designer Module 6: Multimedia and Animations] Typo in the list of applied expertise for Instructors

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wai-curricula#417]
The sixth item reads "WCAG Success Criteria", but it should be "WCAG Success Criterion"

**Comment by daniel-montalvo** [comment]
Thanks for raising this @carlosapaduarte 

Fixed. Will show up when updating the preview.
